test/Microsoft.AspNetCore.Mvc.Abstractions.Test/ModelBinding/ModelStateDictionaryTest.cs

```diff
@@ -757,7 +805,7 @@ public void TryAddModelError_WithErrorString_ReturnsFalse_AndAddsMaxModelErrorMe
         }
 
         [Fact]
-        public void AddModelError_WithException_AddsTooManyModelError_WhenMaxErrorIsReached()
+        public void AddModelError_WithExceptionAndModelMetadata_AddsTooManyModelError_WhenMaxErrorIsReached()
```

Comment:
Might as well revert these name changes (not just this one) since there's no confusion